src/ex4/server/db/migrations/20220622135356-addStatus.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+      await queryInterface.addColumn('Items', 'item_status', {
```

Comment:
I believe you are using this column for storing if the item is done or not. Status is not the right name for this column.